hw/hw4-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+source ../venv/bin/activate
+
+pip install -r ../requirements.txt
+
+rm -rf -- "hw4_plots"
+mkdir -p -- "hw4_plots"
+
+python3 hw4.py
+
+xdg-open ./hw4_plots/results.html
```

Comment:
It seems xdg-open is a Linux command and will not work on OSX. The command for OSX is just 'open,' is there a way to automate this?

Reply:
@jcurry9813 you and colin and your freaking macs, man